go/vt/vtgr/controller/refresh_test.go

```diff
@@ -61,7 +61,7 @@ func TestRefreshTabletsInShard(t *testing.T) {
 	})
 	assert.Equal(t, testHost, instances[0].tablet.Hostname)
 	assert.Equal(t, int32(testPort0), instances[0].tablet.MysqlPort)
-	assert.Equal(t, topodatapb.TabletType_PRIMARY, instances[0].tablet.Type)
+	assert.Equal(t, topodatapb.TabletType_MASTER, instances[0].tablet.Type)
```

Comment:
all master->primary in this file too.. in fact you may be able to simply revert the changes to these two files?

Reply:
good catch, fixed